packages/block-library/src/comment-date/block.json

```diff
@@ -21,7 +21,11 @@
 		"html": false,
 		"color": {
 			"gradients": true,
-			"link": true
+			"link": true,
+			"__experimentalDefaultControls": {
```

Comment:
Same concern as for the author name: should we add link as one of the defaults, given that there's a "Link to comment" setting?